src/controllers/controllerengine.cpp

```diff
@@ -152,10 +152,6 @@ Output:  -
 void ControllerEngine::gracefulShutdown() {
     qDebug() << "ControllerEngine shutting down...";
 
-    // Clear the m_connectedControls hash so we stop responding
-    // to signals.
-    m_connectedControls.clear();
```

Comment:
This did not have any practical effect because the connections are stored in ControlObjectScripts, so I removed this unnecessary member variable.